monitoring/uss_qualifier/scd/data_interfaces.py

```diff
@@ -16,16 +34,34 @@ class SCDTestIssue(ImplicitDict):
     details: str
     '''Human-readable description of the issue'''
 
-    def __init__(self, **kwargs):
-        super(SCDTestIssue, self).__init__(**kwargs)
-        if 'timestamp' not in kwargs:
-            self.timestamp = datetime.datetime.utcnow().isoformat()
+    queries: List[fetch.Query]
+    """Description of HTTP requests relevant to this issue"""
 
 class RequiredResult(ImplicitDict):
     ''' A class to evaluate results / response to an injection of test flight data (TestFlightRequest) '''
-    expected_result: str
-    incorrect_result_details: Optional[SCDTestIssue]
+    acceptable_results: List[str] # Holds information about the "result" response provided by the test interface of the USS under test. Per the SCD Testing API this should be one of "Planned", "Rejected", "ConflictWithFlight" or "Failed"
+    incorrect_result_details: Optional[Dict[Result, KnownIssueFields]] # In the case where the USS under test provides a result that is not the same as expected result then the test driver can provide a message detailing why the expected result was populated the way it was. This should help the USS under test debug specific parts of their internal systems. 
+
+class InjectionTarget(ImplicitDict):
```

Comment:
Let's make a clear section divider between 1) data types used in the interface between the simulator (or other test definition/data generation tool) and the text executor and 2) data types used in the interface between the user and the test executor.  Those two interfaces are very different -- we shouldn't have data types from interface 2 (InjectionTarget + SCDTestConfiguration) interspersed with data types for interface 1.

Reply:
Thanks I have now added the separators.